src/main/java/roomescape/service/ReservationService.java

```diff
@@ -0,0 +1,49 @@
+package roomescape.service;
+
+import java.util.List;
+import org.springframework.stereotype.Service;
+import roomescape.db.ReservationDao;
+import roomescape.db.ReservationTimeDao;
+import roomescape.domain.Reservation;
+import roomescape.dto.ReservationRequest;
+
+
+@Service
+public class ReservationService {
+    private final ReservationDao reservationDao;
+
+    private final ReservationTimeDao reservationTimeDao;
+
+    public ReservationService(
+            final ReservationDao reservationDao,
+            final ReservationTimeDao reservationTimeDao
+    ) {
+        this.reservationDao = reservationDao;
+        this.reservationTimeDao = reservationTimeDao;
+    }
+
+    public Reservation create(final ReservationRequest reservationRequest) {
+        validateNotNull(reservationRequest);
+        final Reservation reservation = new Reservation(
+                reservationRequest.name(),
+                reservationRequest.date(),
+                reservationTimeDao.findById(reservationRequest.timeId())
+                        .orElseThrow(() -> new IllegalArgumentException("존재하지 않는 time id입니다."))
+        );
```

Comment:
```suggestion
        ReservationTime reservationTime = reservationTimeDao.findById(reservationRequest.timeId())
            .orElseThrow(() -> new IllegalArgumentException("존재하지 않는 time id입니다."));
        
        final Reservation reservation = new Reservation(
            reservationRequest.name(),
            reservationRequest.date(), 
            reservationTime
        );
```
적절히 변수를 빼주는게 더 가독성이 좋을것 같아요